kde-base/kdepim-meta/kdepim-meta-4.9999.ebuild

```diff
@@ -31,10 +31,8 @@ RDEPEND="
 	$(add_kdebase_dep korganizer)
 	$(add_kdebase_dep ktimetracker)
 	$(add_kdebase_dep ktnef)
-	nls? ( || (
-			(	$(add_kdebase_dep kde-l10n '' 4.14.3)
-				$(add_kdebase_dep kdepim-l10n '' 4.14.3)	)
-			(	$(add_kdeapps_dep kde4-l10n '' 15.04.0)	)
-		)
+	nls? (
```

Comment:
kde-apps/kde4-l10n blocks kde-apps/kdepim-l10n, so the || () block needs to stay.